extensions/wikia/AdEngine/js/AdLogicPageDimensions.js

```diff
@@ -77,6 +77,7 @@ define('ext.wikia.adEngine.adLogicPageDimensions', [
 			mediaQueriesToCheck.twoColumns = 'screen and (min-width: 1064px)';
```

Comment:
jQuery is not included here. What is more I remember there was an experiment for running Ads in Head and as far as I understand adding jQuery as a dependency wouldn't be optimal solution